packages/form-helpers/README.md

```diff
@@ -0,0 +1,118 @@
+# @open-wc/form-utils
+A collection of form control related utilities for working with forms.
+
+
+## Install
+
+```sh
+# npm
+npm install @open-wc/form-helpers
+
+# yarn
+yarn add @open-wc/form-helpers
+```
+
+### Implicit form submit
+The `submit` helper is a useful helper for firing the forms `submit` event – as a preventable event – only when the form's validity reports back as truthy (meaning the form has all valid values in its inputs) and calling the provided form's `submit()` method if the submit event is not `defaultPrevented`.
```

Comment:
It might be worth calling out that this is somewhat of a stopgap until Safari implements `HTMLFormElement.prototype.requestSubmit` …